LiskHQ/lisk-desktop
# Validator's name shouldn't appear on the wallet page

### What was the problem?

This PR resolves #5126

### How was it solved?

- Remove use of validator name on account page
- Show validator name on my validator profile

### How was it tested?

1. Go to http://0.0.0.0:8080/#/wallet with a validator account 
2. Expected: Account name should be shown and not validator name
3. Go to http://0.0.0.0:8080/#/validators/profile?address=lskguo9kqnea2zsfo3a6qppozsxsg92nuuma3p7ad with a validator account 
4. Expected: Show validator name in the details section
